conda_libmamba_solver/solver.py

```diff
@@ -239,6 +264,68 @@ def __init__(
             fixed_specs.append(spec)
         self.specs_to_add = frozenset(MatchSpec.merge(s for s in fixed_specs))
 
+    def _notify_conda_outdated(
+        self,
+        link_precs,
+        index: LibMambaIndexHelper = None,
+        final_state: Iterable[PackageRecord] = None,
+    ):
+        """
+        We are overriding the base class implementation, which gets called in
+        Solver.solve_for_diff() once 'link_precs' is available. However, we
+        are going to call it before (in .solve_final_state(), right after the solve).
+        That way we can reuse the SolverInputState instance we have lying around,
+        which contains the index information we need, before we lose it.
+        """
+        if index is None and final_state is None:
+            return
+        if not context.notify_outdated_conda or context.quiet:
+            return
+        current_conda_prefix_rec = PrefixData(context.conda_prefix).get("conda", None)
+        if not current_conda_prefix_rec:
+            return
+
+        channel_name = current_conda_prefix_rec.channel.canonical_name
+        if channel_name == UNKNOWN_CHANNEL:
+            channel_name = "defaults"
+        # only look for a newer conda in the channel conda is currently installed from
+        conda_newer_str = f"{channel_name}::conda>4.10"
```

Comment:
Could you elaborate why >4.10 is used here?

Reply:
Oops, I think it was me debugging stuff. It should use the currently installed version.